pkg/syncer/metadatasyncer.go

```diff
@@ -886,7 +886,7 @@ func csiPVCUpdated(ctx context.Context, pvc *v1.PersistentVolumeClaim, pv *v1.Pe
 				VolumeIds: []cnstypes.CnsVolumeId{{Id: volumeHandle}},
 			}
 			// Query with empty selection. CNS returns only the volume ID from it's cache.
```

Comment:
Does this comment still hold true that only volume ID will be returned? Here and at other places.

Reply:
Yes